docs/userguides/testing.md

```diff
@@ -0,0 +1,20 @@
+### Testing
```

Comment:
There is a big section in the `project` guide that covers all of this and more.

If we want this guide to be separate (which I think is probably a good idea), let's keep a section in the `project` guide for testing but just have it link to this guide. Then, move everything in the testing section of the `project` guide to this guide.

Reply:
I concur with this. Testing should be a much richer section.